Report/ResultsUtil.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
```

Comment:
nit- Unused imports, missing license banner. Please review the other files for the same 😃